api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java

```diff
@@ -143,6 +148,26 @@ public TensorsInfo getOutputInfo() {
         return info;
     }
 
+    /**
+     * Sets the maximum amount of time to wait for an output, in milliseconds.
+     *
+     * @param timeout The time to wait for an output
+     *
+     * @throws IllegalArgumentException if given param is invalid
+     * @throws IllegalStateException if failed to set the timeout
+     */
+    public void setTimeout(int timeout) {
```

Comment:
Timeout was unsigned-int in C-API.

Reply:
Because java does not support unsigned type, I added an exception case in this method when timeout is negative value.